python/SWESimulators/Common.py

```diff
@@ -168,40 +165,37 @@ def release(self):
         Frees the allocated memory buffers on the GPU 
         """
         if self.holds_data:
-            self.data.release()
+            del self.data
             self.holds_data = False
 
     
     @staticmethod
     def convert_to_float32(data):
         """
-        Converts to C-style float 32 array suitable for the GPU/OpenCL
+        Converts to C-style float 32 array suitable for the GPU/CUDA
         """
         if (not np.issubdtype(data.dtype, np.float32) or np.isfortran(data)):
-            #print "Converting H0"
```

Comment:
Perhaps this should be kept as a warning, so that we know if something converts (expensive)